### PR TITLE
feat(tokens): add status colors

### PR DESCRIPTION
**Pull request by franckgaudin**

### CODE REVIEW

File: packages/tokens/src/tokens/semantic/light/colors.tokens.json [hunk 1]
```diff
@@ -672,5 +672,187 @@
             "$type": "color",
             "$value": "{coastal.100}"
         }
+    },
+    "status": {
+        "neutral": {
```
Comment (fraincs) [inline packages/tokens/src/tokens/semantic/light/colors.tokens.json:677]: is there a reason why we don't always have the same order in the exports? 

```
"surface-strong"
"text"
"icon"
"surface"
```

vs

```
"surface-strong"
"icon"
"surface"
"text"
```

Comment (fraincs) [reply packages/tokens/src/tokens/semantic/light/colors.tokens.json:677]: I guess we could easily reorder this using AI to make sure we follow the same logic kinda like the Figma file is always using the same order.

Comment (franckgaudin) [reply packages/tokens/src/tokens/semantic/light/colors.tokens.json:677]: I've taken over Figma's export, so I guess it's the order in which the variables are created.

Comment (fraincs) [reply packages/tokens/src/tokens/semantic/light/colors.tokens.json:677]: That was my second question. In that case maybe there's nothing we can easily do. If we don't edit this document by hand ever it's less of a concern.

Comment (franckgaudin) [reply packages/tokens/src/tokens/semantic/light/colors.tokens.json:677]: I find it hard to see the added value of doing this at the moment, as the data coming from Figma has no particular order and will ultimately be converted for the end user.

**Review by fraincs — APPROVED** [review]

**Review by fraincs — APPROVED** [review]

**Pull request merged**